server/konnectors/meetup.js

```diff
@@ -0,0 +1,65 @@
+const request = require('request').defaults({ jar: true });
+const cheerio = require('cheerio');
+const ical = require('./ical_feed');
+
+const baseKonnector = require('../lib/base_konnector');
+
+/**
+ * The goal of this konnector is to fetch the iCal of the user with his Meetup events
+ */
+module.exports = baseKonnector.createNew({
+  name: 'Meetup',
+  vendorLink: 'http://www.meetup.com',
+
+  fields: {
+    login: 'text',
+    password: 'password'
+  },
+
+  models: [],
+  fetchOperations: [
+    login
+  ]
+});
+
+function login(requiredFields, billInfos, data, next) {
+  request('https://secure.meetup.com/login/', (err, res, body) => {
+    if (err) return next(new Error(`Cannot get login page: ${err}`));
```

Comment:
Please log the error and return a localization key instead ("request error" for example).